pkg/cmd/admin/network/isolate_projects.go

```diff
@@ -9,6 +9,8 @@ import (
 	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	kerrors "k8s.io/kubernetes/pkg/util/errors"
 
+	osdnapi "github.com/openshift/openshift-sdn/plugins/osdn/api"
```

Comment:
OK then, for now I'm going to just revert the pkg/cmd/ part of the change; we already have origin/pkg/sdn/api and openshift-sdn/plugins/osdn/api, and the latter already depends on random stuff from kubernetes that we presumably don't want "oc" depending on, so it can't be moved wholesale to openshift-sdn/pkg/. So that means we would need some third "api" directory in openshift-sdn/pkg/ with some slightly different name, which will just be confusing.

We can revisit this in a later cleanup.